zero-phoenix/README.md

```diff
@@ -0,0 +1,456 @@
+# Phoenix Example
+
+The purpose of this example is to provide details as to how one would go about using GraphQL with the Phoenix Web Framework.  Thus, I have created two major sections which should be self explanatory: Quick Installation and Tutorial Installation.
+
+## Getting Started
+
+## Software requirements
+
+- [Elixir 1.4.0 or higher](http://elixir-lang.org/install.html)
```

Comment:
I think this is wrong. You have 1.5.0-dev specified in your `mix.exs` file.

I was able to push on through by using `brew install elixir --HEAD`.

Feel free to send a follow up PR correcting this!